### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by fsocie7y**

### CODE REVIEW

**Review by rostyak — APPROVED** [review]

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+            id="animal age less than 15"
+        ),
+        pytest.param(
+            16, 19, [1, 1],
+            id="animal age is more 15 and less 23"
+        ),
+        pytest.param(
+            23.9, 23.1, [1, 1],
+            id="animal age with remainder"
+        ),
```
Comment (TPolina) [inline app/test_main.py:24]: you need to test 14, 15, 23, 24 years for both cats and dogs, because these values are edge cases

**Review by TPolina — CHANGES_REQUESTED** [review]
